### DISCUSSION TITLE
When I tried to convert the DRIVE dataset, I encountered a bug

### DISCUSSION THREAD

**Discussion by MrWcy** [open-mmlab/mmsegmentation#2461] [General]
I converted the format of retinal segmentation dataset according to the tutorial. All datasets except DRIVE were successful. I encountered a bug. I need this dataset urgently now. Do you have a converted DRIVE dataset and can you send it to my email 735746916@qq.com , thank you.

**Comment by xiexinch** [comment]
Hi @MrWcy,
Could you provide more information about the bug?

**Reply by MrWcy** [reply]
`Cloning into 'datasets'...
remote: Enumerating objects: 125, done.
remote: Counting objects: 100% (125/125), done.
remote: Compressing objects: 100% (119/119), done.
remote: Total 125 (delta 7), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (125/125), 57.06 MiB | 11.13 MiB/s, done.
Resolving deltas: 100% (7/7), done.
/opt/conda/lib/python3.7/site-packages/mmcv/__init__.py:21: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  'On January 1, 2023, MMCV will release v2.0.0, in which it will remove '
Making directories...
Extracting training.zip...
Generating training dataset...
Traceback (most recent call last):
  File "tools/convert_datasets/drive.py", line 113, in <module>
    main()
  File "tools/convert_datasets/drive.py", line 50, in main
    for img_name in os.listdir(now_dir):
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp5wtu8q59/training/images'`

**Reply by MrWcy** [reply]
DRIVE数据集的官网缺少1st_manual文件夹，当我把这个1st_manual放到test.zip中的时候就会出现这个错误，但是没有1st_manual训练过程中又没办法评估
